docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+if [ -e /rttys/rttys.conf ]; then
+    # randomize rttys http username and password
+    USERNAME=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 20)
+    PASSWORD=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 20)
+    sed "s/http-username:.*/http-username: ${USERNAME}/g" /rttys/rttys.conf > tmp && cat tmp > /rttys/rttys.conf
+    sed "s/http-password:.*/http-password: ${PASSWORD}/g" /rttys/rttys.conf > tmp && cat tmp > /rttys/rttys.conf
```

Comment:
Based on the latest changes in https://github.com/zhaojh329/rttys/tree/v3.7.0 , it doesn't seem to matter what are the values of `http-*` values. Have you added it for the backward compatibility with 3.5.0?

Reply:
You are right, probably cause they added the user signup functionality. This probably means that there are no valid credentials initially but anyone can create an account, right?.

Reply:
Probably so